packages/shared/lib/Cargo.toml

```diff
@@ -16,15 +16,15 @@ dev = []
 [dependencies]
 async-trait = {version = "0.1.51"}
 tiny-bip39 = "0.8.2"
-borsh = {version = "1.0.0-alpha.4", features = ["schema", "derive"]}
-borsh-ext = {tag = "v1.0.0-alpha.4", git = "https://github.com/heliaxdev/borsh-ext"}
+borsh = {version = "1.2.0", features = ["unstable__schema", "derive"]}
+borsh-ext = { git = "https://github.com/heliaxdev/borsh-ext", tag = "v1.2.0" }
```

Comment:
We should change borsh deps and masp_primitives in crypto package also, just to be sure :) I've noticed that when I was trying to reduce bundles size.

Reply:
Done!